platform/wasm/Custom-wasm.cmake

```diff
@@ -0,0 +1,30 @@
+set(USE_PORTAUDIO 0)
+set(USE_PORTMIDI 0)
+set(HAVE_SPRINTF_L 0)
+set(USE_IPMIDI 0)
+set(BUILD_TESTS 0)
+set(BUILD_STATIC_LIBRARY 1)
+set(BUILD_JAVA_INTERFACE OFF)
+set(INSTALL_PYTHON_INTERFACE OFF)
+set(BUILD_UTILITIES OFF)
+set(USE_DOUBLE 0)
+
+set(BUILD_CSBEATS ON)
+set(BUILD_CSOUND_COMMAND ON)
+
+
+if(${CMAKE_BUILD_TYPE} MATCHES "Release")
+    set(CMAKE_C_FLAGS "-fno-exceptions -sUSE_PTHREADS=1 -sSUPPORT_LONGJMP='wasm' -sWASM_BIGINT -Wall -fPIC")
```

Comment:
Shouldn't this include `CMAKE_C_FLAGS` as well? Otherwise you're just overwriting it

Reply:
I updated the pr to include CMAKE_C_FLAGS.